### PR TITLE
Add hidden blamer watcher

### PR DESCRIPTION
**Pull request by jakubdybczak**

### CODE REVIEW

**Comment by piaskowyk** [comment]
Good job my bro !

File: src/watcher.py [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+from src.watcher_services.hidden_blamer import HiddenBlamer
 
 
 class Watcher:
     trigger = Trigger()
+    hidden_blamer = HiddenBlamer()
 
     async def check(self, message):
-        await self.trigger.run(message)
+        # await self.trigger.run(message)
```
Comment (matix522) [inline src/watcher.py:10]: dead code

**Comment by Sewerinius** [comment]
Worst code I've ever seen. Approved.

File: src/watcher_services/hidden_blamer.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+from src.utils.utils import rand_item
+
+
+class HiddenBlamer:
+
+    responses = [
+        lambda author: f"Niewidoczni, tacy jak <@!{author.id}> nie mają prawa głosu!",
+        lambda author: f"<@!{author.id}>, proszę zmienić status na online",
+        lambda author: "Niewidoczni nie mają prawa głosu!",
+        lambda author: "Niewidoczni nie mają racji!",
```
Comment (piaskowyk) [inline src/watcher_services/hidden_blamer.py:11]: True strory!

File: src/watcher_services/hidden_blamer.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import discord
+from src.utils.utils import rand_item
+
+
+class HiddenBlamer:
+
+    responses = [
+        lambda author: f"Niewidoczni, tacy jak <@!{author.id}> nie mają prawa głosu!",
```
Comment (matix522) [inline src/watcher_services/hidden_blamer.py:8]: Could maybe be list of strings mapped to become lambda using
`
map(lambda string: lambda: string)
`

**Review by matix522 — CHANGES_REQUESTED** [review]

File: src/watcher.py (hunk 1 above)
Comment (piaskowyk) [reply src/watcher.py:10]: Like me insede (:

Comment (jakubdybczak) [reply src/watcher.py:10]: @piaskowyk should we delete this?

Comment (Sewerinius) [reply src/watcher.py:10]: Wake me up

**Review by piaskowyk — APPROVED** [review]

File: src/watcher_services/hidden_blamer.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        lambda author: "Ej nie słyszymy!",
+        lambda author: "Oj, ktoś coś mówił?",
+        lambda author: "Chyba się przesłyszałem",
+        lambda author: "Ktoś coś mówił?",
+        lambda author: "Halo, ktoś coś mówił?",
+        lambda author: "Kto nie słyszy?",
+        lambda author: "Halo?!",
+        lambda author: "Czy ktoś z państwa słyszał coś?",
+        lambda author: "Ludzie, przecież tu nikogo nie ma"
+    ]
```
Comment (Sewerinius) [inline src/watcher_services/hidden_blamer.py:26]:
```python
def getResponces(author):
    return [ ... ]
```

Comment (Sewerinius) [reply src/watcher_services/hidden_blamer.py:26]: Or maybe even 
`responces = lambda author : [ ... ]`

**Review by Sewerinius — APPROVED** [review]

**Pull request merged**